tools/nni_annotation/README.md

```diff
@@ -83,10 +83,10 @@ h_pooling = max_pool(hidden_layer, pool_size)
 
    `'''@nni.report_intermediate_result(metrics)'''`
 
-`@nni.report_intermediate_result` is used to report intermediate result, whose usage is the same as `nni.report_intermediate_result` in [Trials.md](https://nni.readthedocs.io/en/latest/Trials.html)
+`@nni.report_intermediate_result` is used to report intermediate result, whose usage is the same as `nni.report_intermediate_result` in [Trials.md](https://nni.readthedocs.io/en/latest/TrialExample/Trials.html)
 
 ### 4. Annotate final result
 
    `'''@nni.report_final_result(metrics)'''`
 
-`@nni.report_final_result` is used to report the final result of the current trial, whose usage is the same as `nni.report_final_result` in [Trials.md](https://nni.readthedocs.io/en/latest/Trials.html)
+`@nni.report_final_result` is used to report the final result of the current trial, whose usage is the same as `nni.report_final_result` in [Trials.md](https://nni.readthedocs.io/en/latest/TrialExample/Trials.html)
```

Comment:
in [Trials.md] -> in the doc of [Write a trial run on NNI]

Reply:
fixed